store/s3_test.go

```diff
@@ -1,23 +1,51 @@
 package store_test
 
 import (
+	"crypto/rand"
+	"github.com/medibloc/panacea-data-market-validator/config"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"testing"
 
-	awsendpoints "github.com/aws/aws-sdk-go/aws/endpoints"
 	"github.com/stretchr/testify/require"
 
 	"github.com/medibloc/panacea-data-market-validator/store"
 )
 
+func TestStorageUpload(t *testing.T) {
+	conf := &config.Config{
+		AWSS3: config.AWSS3Config{
+			Region:          "ap-northeast-2",
+			Bucket:          "data-market-test",
+			AccessKeyID:     os.Getenv("EDG_DATAVAL_AWS_S3_ACCESS_KEY_ID"),
+			SecretAccessKey: os.Getenv("EDG_DATAVAL_AWS_S3_SECRET_ACCESS_KEY")},
+	}
+
+	s3Store, err := store.NewS3Store(conf)
+	require.NoError(t, err)
+
+	path := "temp_path"
+	name := "name"
+
+	data, err := randomBytes(100000)
```

Comment:
You can use the `crypto.GenerateRandomKey()` :)

Reply:
Oh, Thanks. I'll fix it.